src/js/requestAwareMiddleware.js

```diff
@@ -0,0 +1,45 @@
+/*
+
+    Middleware that creates a `gpii.express.handler` component and uses that to process each request.  See the
+    documentation for details:
+
+    https://github.com/GPII/gpii-express/blob/master/docs/middleware.md
+
+ */
+"use strict";
+var fluid = require("infusion");
+
+fluid.defaults("gpii.express.middleware.requestAware", {
+    gradeNames: ["gpii.express.middleware"],
+    timeout: 5000, // The default timeout we will pass to whatever grade we instantiate.
+    distributeOptions: [
+        {
+            source: "{that}.options.timeout",
+            target: "{that gpii.express.handler}.options.timeout"
+        },
+        {
+            source: "{that}.options.handlerGrades",
+            target: "{that gpii.express.handler}.options.gradeNames"
+        }
+    ],
+    method:     "get",
```

Comment:
The entire line was left over from the previous implementation, and caused problems elsewhere.  I have removed it entirely, the grade will now correctly pick up the default `use` method from `gpii.express.middleware`.